src/components/calendar/DayCard.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex, Spacer, HStack, Badge, Text, SlideFade } from '@chakra-ui/react';
+import dayjs from 'dayjs';
+
+interface Props {
+  day: string;
+  monthOffset: number;
+  direction: number;
+}
+
+export function DayCard(props: Props) {
+  const currentMonth = dayjs().add(props.monthOffset, 'month').format('MM');
+  const monthThisCardIsIn = dayjs(props.day).format('MM');
+  const isInCurrentMonth = currentMonth === monthThisCardIsIn;
```

Comment:
Always nice to see types, if not for _currentMonth_/_monthThisCardIsIn_ (btw. change this to _cardMonth_?) then at least mark isInCurrentMonth as bool.